tst/EnergyPlus/unit/SurfaceGeometry.unit.cc

```diff
@@ -8554,3 +8554,192 @@ TEST_F(EnergyPlusFixture, Use_Gross_Roof_Area_for_Averge_Height_with_Window)
     EXPECT_NEAR(state->dataHeatBal->Zone(1).CeilingHeight, ceilingHeight_expected, 1e-6);
     EXPECT_NE(state->dataHeatBal->Zone(1).CeilingHeight, ceilingHeight_old);
 }
+TEST_F(EnergyPlusFixture, SurfaceGeometry_ZoneAndSpaceAreas)
```

Comment:
New unit test to cover some combinations of space and/or zone floor area inputs.